modules/ding_frontpage/ding_frontpage.pages_default.inc

```diff
@@ -119,62 +119,6 @@ function ding_frontpage_default_page_manager_pages() {
     $display->content['new-18cfc092-1d0f-42e4-bde8-8a6e314c9bd0'] = $pane;
     $display->panels['primary'][1] = 'new-18cfc092-1d0f-42e4-bde8-8a6e314c9bd0';
     $pane = new stdClass();
-    $pane->pid = 'new-3f5761b9-1db4-4dc9-87cb-13e33008a1fc';
-    $pane->panel = 'primary';
-    $pane->type = 'interaction_pane';
-    $pane->subtype = 'interaction_pane';
-    $pane->shown = TRUE;
-    $pane->access = array();
-    $pane->configuration = array(
-      'count' => '5',
-      'override_title' => 0,
-      'override_title_text' => '',
-      'override_title_heading' => 'h2',
-    );
-    $pane->cache = array();
-    $pane->style = array(
-      'settings' => NULL,
-    );
-    $pane->css = array();
-    $pane->extras = array();
-    $pane->position = 2;
-    $pane->locks = array();
-    $pane->uuid = '3f5761b9-1db4-4dc9-87cb-13e33008a1fc';
-    $display->content['new-3f5761b9-1db4-4dc9-87cb-13e33008a1fc'] = $pane;
-    $display->panels['primary'][2] = 'new-3f5761b9-1db4-4dc9-87cb-13e33008a1fc';
-    $pane = new stdClass();
-    $pane->pid = 'new-5e2a0d2d-7b27-498e-a368-d4e9a2888769';
-    $pane->panel = 'primary';
-    $pane->type = 'campaign';
-    $pane->subtype = 'campaign';
```

Comment:
Do we really want to remove this ding campaign panel pane?

Reply:
Yeah, it's old cruft from the old campain module. When campaign_plus took over, nobody bothered to update all the panels, so there's missing panes floating about.